tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -287,46 +287,49 @@ impl CompressedCiphertextList {
     ///
-    ///         let compressed = CompressedCiphertextListBuilder::new()
-    ///             .push(ct1)
-    ///             .push(ct2)
-    ///             .push(ct3)
-    ///             .build(&compression_key);
+    ///     let compressed = CompressedCiphertextListBuilder::new()
+    ///         .push(ct1)
+    ///         .push(ct2)
+    ///         .push(ct3)
+    ///         .build(&compression_key);
     ///
-    ///         let cuda_compressed = compressed.to_cuda_compressed_ciphertext_list(&streams);
+    ///     let cuda_compressed = compressed.to_cuda_compressed_ciphertext_list(&streams);
+    ///     let recovered_cuda_compressed = cuda_compressed.to_compressed_ciphertext_list(&streams);
     ///
-    ///         let d_decompressed1: CudaUnsignedRadixCiphertext =
-    ///             cuda_compressed.get(0, &cuda_decompression_key, &streams).unwrap().unwrap();
-    ///         let decompressed1 = d_decompressed1.to_radix_ciphertext(&streams);
-    ///         let decrypted: u32 = radix_cks.decrypt(&decompressed1);
-    ///         assert_eq!(decrypted, 3_u32);
+    ///    assert_eq!(recovered_cuda_compressed, compressed);
     ///
-    ///         let d_decompressed2: CudaSignedRadixCiphertext =
-    ///             cuda_compressed.get(1, &cuda_decompression_key, &streams).unwrap().unwrap();
-    ///         let decompressed2 = d_decompressed2.to_signed_radix_ciphertext(&streams);
-    ///         let decrypted: i32 = radix_cks.decrypt_signed(&decompressed2);
-    ///         assert_eq!(decrypted, -2);
+    ///     let d_decompressed1: CudaUnsignedRadixCiphertext =
+    ///         cuda_compressed.get(0, &cuda_decompression_key, &streams).unwrap().unwrap();
+    ///     let decompressed1 = d_decompressed1.to_radix_ciphertext(&streams);
+    ///     let decrypted: u32 = radix_cks.decrypt(&decompressed1);
+    ///     assert_eq!(decrypted, 3_u32);
     ///
-    ///         let d_decompressed3: CudaBooleanBlock =
-    ///             cuda_compressed.get(2, &cuda_decompression_key, &streams).unwrap().unwrap();
-    ///         let decompressed3 = d_decompressed3.to_boolean_block(&streams);
-    ///         let decrypted = radix_cks.decrypt_bool(&decompressed3);
-    ///         assert!(decrypted);
+    ///     let d_decompressed2: CudaSignedRadixCiphertext =
+    ///         cuda_compressed.get(1, &cuda_decompression_key, &streams).unwrap().unwrap();
+    ///     let decompressed2 = d_decompressed2.to_signed_radix_ciphertext(&streams);
+    ///     let decrypted: i32 = radix_cks.decrypt_signed(&decompressed2);
+    ///     assert_eq!(decrypted, -2);
+    ///
+    ///     let d_decompressed3: CudaBooleanBlock =
+    ///         cuda_compressed.get(2, &cuda_decompression_key, &streams).unwrap().unwrap();
+    ///     let decompressed3 = d_decompressed3.to_boolean_block(&streams);
+    ///     let decrypted = radix_cks.decrypt_bool(&decompressed3);
+    ///     assert!(decrypted);
```

Comment:
why the indentation here ?

Reply:
For some reason `make fmt_gpu` will not correctly indent this part. I tried to fix indentation manually.

Reply:
still broken it seems, I'll try to see if I can fix it

Reply:
I fixed it manually, but this file doesn't seem to be formatted at all when doing make fmt_gpu, this is strange.

Reply:
does it have a very long line ? rust fmt sometimes has issues with taht